knative-operator/hack/update-deps.sh

```diff
@@ -13,4 +13,8 @@ go mod vendor
 # TODO: Remove this once we bump kubernetes versions.
 git apply "$ROOT_DIR/hack/manifestival.patch"
 
-find vendor/ \( -name "OWNERS" -o -name "OWNERS_ALIASES" -o -name "BUILD" -o -name "BUILD.bazel" -o -name "*_test.go" \) -print0 | xargs -0 rm -f
\ No newline at end of file
+find vendor/ \( -name "OWNERS" \
+  -o -name "OWNERS_ALIASES" \
+  -o -name "BUILD" \
+  -o -name "BUILD.bazel" \
+  -o -name "*_test.go" \) -print0 | xargs -0 rm -f
```

Comment:
Hey. I've putted that command differentlyin my review.

It was

```suggestion
  -o -name "*_test.go" \) | xargs -r rm -fv
```

Reply:
What's the significance of this?

Reply:
shellcheck barks at me if I drop the `-print0` thing.

Reply:
Right. Let's go with `find ... -exec rm -fv {} +`

Reply:
Done.